libs/agora/ui/src/lib/footer/footer.component.ts

```diff
@@ -9,7 +14,44 @@ import { RouterModule } from '@angular/router';
   templateUrl: './footer.component.html',
   styleUrls: ['./footer.component.scss'],
 })
-export class FooterComponent {
-  @Input({ required: true }) appVersion = '';
-  @Input({ required: true }) apiDocsUrl = '';
+export class FooterComponent implements OnInit {
+  footerLogoPath!: SafeUrl;
+  dataVersion$!: Observable<Dataversion>;
+
+  navItems: Array<any> = [
```

Comment:
Consider defining an interface of type in this file instead of using `any`.

For example:

```typescript
type Link = {
  label: string;
  url: string;
  target?: '_blank' | '_self' | '_parent' | '_top';
};
```

Reply:
This definitely can be defined.  Done!